TestData/UpfileTest/UpfileOverride.xml

```diff
@@ -1,5 +1,5 @@
-<UpfileOverride>
+<DotUplift>
```

Comment:
Do we need to provide a migration for that?

Reply:
BTW, we call this DotUplift in XML,  DotUpfile in code, and UpfileOverride.xml in test....

I would prefer a file called maybe named UpliftSettings and saved for example under

$HOME/.uplift/settings.xml

or similar.

WDYT ?

Reply:
Good idea. I will do that!